lib/PlaidClient.js

```diff
@@ -116,15 +116,24 @@ Client.prototype.updateItemWebhook =
 // createProcessorToken(String, String, String, Function)
 Client.prototype.createProcessorToken =
   function(access_token, account_id, processor, cb) {
-    var endpoint = processor === 'stripe' ?
-      '/processor/stripe/bank_account_token/create' :
-      '/processor/' + processor + '/processor_token/create';
+    var endpoint = '/processor/token/create';
+    var options = {
```

Comment:
This can be a `const` so we're not replacing the value of `options`.

Reply:
⚡️  i wasnt sure what version of node we were supporting